packages/openops/src/lib/cloud-cli-common.ts

```diff
@@ -22,7 +23,37 @@ export function handleCliError({
     error,
   });
 
-  const message = `An error occurred while running a ${provider} CLI command: ${error}`;
+  const message = `An error occurred while running ${provider} CLI command: ${error}`;
```

Comment:
alright I KNOW this could be a seperate pr but pls dont make me do it. Removed the a bc "a AWS" and "a Azure" command are not grammatically correct